### DISCUSSION TITLE
doris应用proxysql负载均衡后，使用datax进行doriswriter数据写入速度过慢

### DISCUSSION THREAD

**Discussion by zzxVv** [apache/doris#34234] [A - General / Q&A]
未搭建负载均衡前，导入速度均在20M/s，搭载后使用jdbc连接proxysql和fe速度均到不了1M/s
![微信截图_20240428162914](https://github.com/apache/doris/assets/92431478/dfff48d5-f2e1-40b3-bdd5-dbaff7881c3a)

**Comment by yagagagaga** [comment]
If connecting directly to FE is also slow, then it is not caused by proxySQL, it may be caused by other problems.
